pkg/artifacts/artifacts_storage.go

```diff
@@ -59,7 +59,25 @@ func InitializeArtifactStorage(pr *v1alpha1.PipelineRun, c kubernetes.Interface,
 	return NewArtifactBucketConfigFromConfigMap(configMap)
 }
 
-func needsPVC(configMap *corev1.ConfigMap, err error, logger *zap.SugaredLogger) (bool, error) {
+// CleanupArtifactStorage will delete the PipelineRun's PVC if it exists.
```

Comment:
maybe elaborate in the comment here about what a "PipelineRun's PVC" is? (i.e. it's not that _all_ PVCs a Run is using will automatically get cleaned up, it's specifically PVCs that are used for sharing between TaskRuns right?)

Reply:
Done.